src/charm.py

```diff
@@ -137,6 +145,28 @@ def __init__(self, *args: Any) -> None:
 
         self.endpoints_provider = KratosEndpointsProvider(self)
 
+        self.metrics_endpoint = MetricsEndpointProvider(
+            self,
+            relation_name=self._prometheus_scrape_relation_name,
+            jobs=[
+                {
+                    "metrics_path": "/metrics/prometheus",
+                    "static_configs": [
+                        {
+                            "targets": ["*:4434"],
```

Comment:
There's already a variable for `4434` right, we probably want to leverage it. The same applies at a couple of places in this file.